src/Illuminate/Auth/Access/Gate.php

```diff
@@ -162,7 +164,7 @@ public function check($ability, $arguments = [])
      *
      * @param  string  $ability
      * @param  array  $arguments
-     * @return callable
+     * @return mixed
```

Comment:
This return type is either array, either callable - in this case it should be mixed